databases/app-config-sync/fragment-feature-release-toggle-definitions.js

```diff
@@ -0,0 +1,37 @@
+{
+  channels: {
+    view: [ 'view-feature-release-toggle-definitions', 'view-config' ],
+    add: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    replace: [ 'edit-feature-release-toggle-definitions', 'edit-config' ],
+    remove: [ 'remove-feature-release-toggle-definitions', 'remove-config' ]
+  },
+  typeFilter: function(doc, oldDoc) {
+    // Example valid document IDs: "feature-release-toggle.foo-toggle", "feature-release-toggle.bar_1"
+    return /^feature-release-toggle\.[a-z0-9_-]+$/.test(doc._id);
+  },
+  propertyValidators: {
+    name: {
+      type: 'string',
+      required: true,
+      mustNotBeEmpty: true,
+      immutableWhenSet: true,
```

Comment:
This might as well be changed to `immutable` since the value can never be `null` or missing/`undefined` anyway because of the `required` property.